waffle/tests/test_testutils.py

```diff
@@ -3,14 +3,14 @@
 from decimal import Decimal
 
 from django.contrib.auth.models import AnonymousUser
-from django.test import TestCase, RequestFactory
+from django.test import TransactionTestCase, RequestFactory
```

Comment:
Django's `TestCase` class automatically wraps each test method in a transaction and rolls back the transaction after the method returns. Since updating models now doesn't flush the cache until after the transaction is committed, this now breaks the logic in most of these tests. Inheriting from `TransactionTestCase` now addresses the problem by removing Django's magical auto-rollback behavior, the trade-off being that the tests aren't quite as fast now.